docs/NMOS With H.264.md

```diff
@@ -58,11 +58,116 @@ The term 'strict-CBR' corresponds to the definition of constant bit-rate at sect
+
+- [Components](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/#components)
+  The Flow resource MUST indicate the color (sub-)sampling, width, height and depth of the associated uncompressed picture using the `components` attribute. The `components` array values MUST correspond to the stream's active parameter sets values. A Flow MUST track the stream's current active parameter sets.
+
+Informative note: ST 2110-22 does not require the `sampling` or `depth` SDP parameters. RFC 6184 does not define any such SDP parameters. The `sampling` and `depth` of the associated uncompressed picture could be derived from the H.264 active parameter sets by a Receiver.
+
+- [Profile](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/#profile)
+  The Flow resource MUST indicate the H.264 profile, which defines algorithmic features and limits that SHALL be supported by all decoders conforming to that profile. It SHALL comply with the stream's active parameter sets. The permitted `profile` values are strings, defined as per ITU-T  Rec. H.264 Annex A
+
+  - "BaselineConstrained"
+  - "Baseline" (Default if not specified in the SDP transport file)
+  - "Main"
+  - "Extended"
+  - "High"
+  - "HighProgressive"
+  - "HighConstrained"
+  - "High10"
+  - "High10Progressive"
+  - "High-422"
+  - "HighPredictive-444"
+  - "High10Intra"
+  - "HighIntra-422"
+  - "HighIntra-444"
+  - "CAVLCIntra-444"
+
+Informative note: The names of the profiles in string form have been derived from the names used at Annex A of the H.264 standard with whitespace omitted, the sampling mode always positioned at the end of the string, preceded by a '-' and the terms High and Baseline positioned at the beginning of the string.
+
+The [NMOS Parameter Registers] is the official reference for the profiles strings. In the case of discrepancies between this specification and the [NMOS Parameter Registers], the registers have precedence.
+
+- [Level](https://specs.amwa.tv/nmos-parameter-registers/branches/main/flow-attributes/#level)
+  The Flow resource MUST indicate the H.264 level, which defines a set of limits on the values that may be taken by the syntax elements of an H.264 bitstream. It SHALL comply with the stream's active parameter sets. The permitted `level` values are strings, defined as per ITU-T Rec. H.264 Annex A
+
+  - "1" (Default if not specified in the SDP transport file)
+  - "1b", "1.1", "1.2", "1.3"
+  - "2", "2.1", "2.2"
+  - "3", "3.1", "3.2"
+  - "4", "4.1", "4.2"
+  - "5", "5.1", "5.2"
+  - "6", "6.1", "6.2"
+
+Informative note: The names of the levels in string form have been derived from the names used at Annex A of the H.264 standard.
+
+The [NMOS Parameter Registers] is the official reference for the levels strings. In the case of discrepancies between this specification and the [NMOS Parameter Registers], the registers have precedence.
```

Comment:
Missed this in previous review, but for consistency...

```suggestion
The level strings in this specification are included in the [NMOS Parameter Registers][]. Additional strings may be added there in the future.
```

Reply:
done